dev-python/django-tagging/django-tagging-0.4.6.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Generic tagging application for Django"
+HOMEPAGE="https://pypi.org/project/django-tagging/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND=">=dev-python/django-1.0[${PYTHON_USEDEP}]"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/sphinx[${PYTHON_USEDEP}]
+	)
+"
+
+python_prepare_all() {
+	sed -i "/sphinx.ext.intersphinx/d" docs/conf.py || die
+	distutils-r1_python_prepare_all
+}
+
+python_compile_all() {
+	if use doc; then
+		sphinx-build docs docs/_build/html || die
+		HTML_DOCS=( docs/_build/html/. )
+	fi
+}
```

Comment:
Any chance for tests, please? ;-)

Reply:
As I said in the rescription:
> I did not add tests for django-tagging because they fail, even fail upstream. For some reason, the developer did not enable issue reporting on GitHub so I have emailed him about that.

I've received a reply from upstream and they said they would look into it but they don't want to allow issue reporting because it would add too much work :/